<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width> <!-- Increased main window width -->
    <height>1000</height> <!-- Increased main window height -->
   </rect>
  </property>
  <property name="windowTitle">
   <string>Workplace Planner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetItems">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>850</height> <!-- Adjusted height to fit button -->
     </rect>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUploadAssets">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y> <!-- Positioned below the list widget -->
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Assets</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCanvas">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>1100</width> <!-- Increased canvas width -->
      <height>900</height> <!-- Increased canvas height -->
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExport">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>920</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Plan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonImport">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>920</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Plan from XML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDelete">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>920</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selected Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width> <!-- Adjusted width -->
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
